src/commands/add-permission.yml

```diff
@@ -9,19 +9,7 @@ parameters:
 steps:
   - run:
       name: Granting << parameters.permission-type >> permissions to << parameters.bundle-id >>
-      command: |
-          if csrutil status | grep -q 'disabled'; then
-              epochdate=$(($(date +'%s * 1000 + %-N / 1000000')))
-              macos_major_version=$(sw_vers -productVersion | awk -F. '{ print $1 }')
-              if [[ $macos_major_version -le 10 ]]; then
-                  tcc_update="replace into access (service,client,client_type,allowed,prompt_count,indirect_object_identifier,flags,last_modified) values (\"<< parameters.permission-type >>\",\"<< parameters.bundle-id >>\",0,1,1,\"UNUSED\",0,$epochdate);"
-              else
-                  tcc_update="replace into access (service,client,client_type,auth_value,auth_reason,auth_version,indirect_object_identifier,flags,last_modified) values (\"<< parameters.permission-type >>\",\"<< parameters.bundle-id >>\",0,2,1,1,\"UNUSED\",0,$epochdate);"
-              fi
-              sudo sqlite3 "/Library/Application Support/com.apple.TCC/TCC.db" "$tcc_update"
-              sudo sqlite3 "/Users/$USER/Library/Application Support/com.apple.TCC/TCC.db" "$tcc_update"
-          else
-              echo "Unable to add permissions! System Integrity Protection is enabled on this image"
-              echo "Please choose an image with SIP disabled. Documentation: https://circleci.com/docs/2.0/testing-macos"
-              exit 1
-          fi
+      environment:
+        ORB_VAL_BUNDLE_ID: << parameters.bundle-id >>
```

Comment:
Should these be `evaluated` in the `add-permission.sh` script? 

Is there a case where a user can use environment variables in these parameters?

Reply:
@brivu Can you clarify what you mean here? As far as I can tell this should all be fine